Formula/hexo.rb

```diff
@@ -0,0 +1,27 @@
+require "language/node"
+
+class Hexo < Formula
+  desc "A fast, simple & powerful blog framework"
+  homepage "https://hexo.io/"
+  url "https://registry.npmjs.org/hexo/-/hexo-5.2.0.tgz"
+  sha256 "3356e9c498af656e0f4fc34673fb8106b33ac6b0d0565efe00fff796bcadc1c8"
+  license "MIT"
+  head "https://github.com/hexojs/hexo.git"
+
+  livecheck do
+    url :stable
+  end
+
+  depends_on "node"
+
+  def install
+    mkdir_p libexec/"lib"
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    output = shell_output("#{bin}/hexo --help")
+    assert_match "Usage: hexo <command>", output.strip
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Ok, I open a issue for official help.  https://github.com/hexojs/hexo/issues/4570